### PR TITLE
Postgres 3

### PR DESCRIPTION
**Pull request by gigicupp**
@christulin @amosNash 
Hi team,
Can you please do a code review for me? I sat up new relic and artillery on this. Found out that my service doesn't do well anything beyond 300 request per second.

### CODE REVIEW

File: server/database-postgres/helpers.js [hunk 1]
```diff
@@ -80,6 +80,16 @@ const formatData = (userData, languageData) => {
 //     .catch(error => console.log(error))
 // }
 
+//update or create
```
Comment (amosNash) [inline server/database-postgres/helpers.js:83]: can get rid of commented out, unless needed.

File: server/app.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ app.put('/users/:userId', async (req, res) => {
 app.delete('/users/:userId', async (req, res) => {
   let id = req.params.userId;
   try {
-    await deleteById(id)
+    await deleteUserById(id)
```
Comment (amosNash) [inline server/app.js:56]: Nice variable name change!

File: server/database-postgres/helpers.js [hunk 3]
```diff
@@ -80,18 +80,22 @@ const formatData = (userData, languageData) => {
...
-//     .then(result => result)
-//     .catch(error => console.log(error))
-// }
+const deleteUserById = (id) => {
+  User.destroy({ where: { userId: id }})
+    .then(result => result)
+    .catch(error => console.log(error))
+  UserLanguage.destroy({ where: { userUserId: id} })
+    .then(result => result)
+    .catch(error => console.log(error))
```
Comment (amosNash) [inline server/database-postgres/helpers.js:99]: good job handling errors.

File: server/artillery/test.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 config:
   target: "http://localhost:5007"
   phases:
-    - duration: 20
-      arrivalRate: 10
+    - duration: 60
```
Comment (amosNash) [inline server/artillery/test.yml:4]: I'm new to artillery. What is the purpose of the phases's duration and arrivalRate? Why are they set to 60 and 300?

**Review by amosNash — COMMENTED** [review]
Looks good! I'm just getting started with artillery and new relic so I don't have any comments on that.

File: server/database-postgres/helpers.js (hunk 1 above)
Comment (gigicupp) [reply server/database-postgres/helpers.js:83]: i have yet to complete my CRUD :(

File: server/artillery/test.yml (hunk 4 above)
Comment (gigicupp) [reply server/artillery/test.yml:4]: i think you figured it out now that you are done with artillery lol. sorry for the late response

**Pull request merged**